packages/oui-navbar/src/navbar.component.js

```diff
@@ -0,0 +1,17 @@
+import controller from "./navbar.controller";
+import template from "./navbar.html";
+
+export default {
+    bindings: {
+        brand: "<?",
+        universe: "@?",
```

Comment:
This component is already oriented for a use for us (e.g. 'notifications' and 'user' menu).
But I can update it to something like `active` or `activeLink`, because it could set a variant `.oui-navbar-link_active` in `asideLinks`.